src/jmclient/maker.py

```diff
@@ -187,6 +189,16 @@ def verify_unsigned_tx(self, tx, offerinfo):
         """
         tx_utxo_set = set((x.prevout.hash[::-1], x.prevout.n) for x in tx.vin)
 
+        if self.minimum_tx_fee_rate > 1000:
```

Comment:
Right. In my comment [here](https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/721#issuecomment-1824627009) I was thinking that perhaps you could use the same strategy as in `bumpfee.py`, but of course you can't ask the blockchain interface for information about the fee; this is not a broadcast (or even a signed) transaction, yet.
So your logic here makes sense to me.

I wanted to put this code in `on_tx_received` not `verify_unsigned_tx` (I guess because this function is so security critical), but I can see your choice makes sense, because here we have `tx_utxo_set`.